earth_enterprise/src/installer/install_server.sh

```diff
@@ -516,8 +516,12 @@ copy_files_to_target()
   cp -f "$TMPOPENLDAPPATH/ldap.conf.default" "$BASEINSTALLDIR_ETC/openldap"
```

Comment:
Change Copyright at top of file to add `Copyright 2019 Open GEE Contributors`

Reply:
done